cookbook/entities/adding_properties.rst

```diff
@@ -0,0 +1,50 @@
+How to Add Properties to Core Entities
+======================================
+
+OroCRM and OroPlatform come with a lot of entities that you may have to customize to fit your
+application's needs. For example, imagine that you want to store the date your contacts became
+member of your company's partner network.
+
+To achieve this you may want to add a new property ``$partnerSince`` which will hold a ``DateTime``
+instance that reflects the date the contact joined your network. This property is added by writing
+a migration:
+
+.. code-block:: php
```

Comment:
Please add the note that the entity class should be configurable (have @Config annotation) and extendable (the class should extends the special empty extend class)